### PR TITLE
Update nodejs version to latest LTS node-v10.16.2-linux-x64

### PR DESCRIPTION
**Pull request by tmkasun**
## Purpose
Updating the NodeJS version

## Goals

Support `npm ci` which comes with v5.7+

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -142,9 +142,9 @@ chmod a+x /build/software/java/jdk-6u33-linux-x64.bin \
 && rm /build/software/java/jdk-6u33-linux-x64.bin
 
 RUN \
-wget -P /build/software/nodejs https://nodejs.org/dist/v8.8.1/node-v8.8.1-linux-x64.tar.xz \
-&& tar -xvf /build/software/nodejs/node-v8.8.1-linux-x64.tar.xz --directory /build/software/nodejs \
-&& rm /build/software/nodejs/node-v8.8.1-linux-x64.tar.xz
+wget -P /build/software/nodejs https://nodejs.org/dist/v10.16.2/node-v10.16.2-linux-x64.tar.xz \
```
Comment (kasunbg) [inline Dockerfile:148]: Anyone else still depend on nodejs 8 for builds? 

cc @mohanvive @jeradrutnam @lalaji

**Review by kasunbg — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (maheshika) [reply Dockerfile:148]: There are other jobs using nodejs 8. Hence, we can't remove the older version.

Comment (mohanvive) [reply Dockerfile:148]: @sajithshn  @CharukaK @lasanthaS Please check & validate

**Comment by lasanthaS** [comment]
We are checking the current ReactJS apps with this node version. Please hold this PR until we provide an update.

File: Dockerfile (hunk 1 above)
Comment (tmkasun) [reply Dockerfile:148]: @maheshika , I have updated the PR to include both versions(v8 & v10) of NodeJS. Hope we could switch to required NodeJS version from Jenkins configuration by specifying the respective paths?

**Comment by lasanthaS** [comment]
We have checked our SP repos with Node 10.16.2, and they are working fine with that version. So +1 on merging this PR.

File: Dockerfile (hunk 1 above)
Comment (maheshika) [reply Dockerfile:148]: I have updated the Jenkins master. Will proceed with building the docker image and configure the respective build.

**Pull request merged**